xcore/interface/stitcher.cpp

```diff
@@ -629,22 +629,39 @@ BowlModel::get_bowlview_vertex_model (
                 bowl_view_image_to_world (
                     _config, _bowl_img_width, _bowl_img_height, texture_pos0);
 
-            vertices.push_back (PointFloat3(world_pos0.x / _config.a, world_pos0.y / _config.b, world_pos0.z / _config.c));
-            indeices.push_back (indicator++);
-            texture_points.push_back (PointFloat2(texture_pos0.x / _bowl_img_width, (_bowl_img_height - texture_pos0.y) / _bowl_img_height));
+            if (world_pos0.z / _config.c < view_height) {
+                if (top_view) {
+                    world_pos0.z = 0.0f;
+                }
+                vertices.push_back (PointFloat3(world_pos0.x / _config.a, world_pos0.y / _config.b, world_pos0.z / _config.c));
```

Comment:
one more.
max(world_pos0.z / _config.c) maybe larger than 1.0f or very smaller than 1.0f. for the bowl_model, maybe
max(world_pos0.z / _config.wall_height) is more precise. this can keep all code in range of [0.0f, 1.0f]